.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "_includes"]
 	path = _includes
-	url = https://github.com/jmapio/jmap.git
+	url = https://github.com/yvesdaxmaz/jmap.git
```

Comment:
This must not be changed.

Reply:
I did that to have hand on both jmapio.github.io project and jmap.io on within and same directory one as a parent and another as a submodule. Like later on I could have to made some changes on the jmap.io repository